packages/web-components/fast-foundation/src/option/option.options.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Option role.
+ * @public
+ */
+export enum OptionRole {
```

Comment:
I don't see this used (though I could be missing it) - can it be removed?

Reply:
Do options have roles, if so, maybe it's just missing the example/unit test?

Reply:
I'd actually prefer we stay away from an enum here personally for option. I see this being used for checking the value in certain places, but enums can be costly as they grow over time and an option *only ever* should have a role of option. If we can remove this I think it would be better.